ui/app/AppLayouts/Chat/panels/communities/TokenHoldersPanel.qml

```diff
@@ -15,10 +15,14 @@ import shared.controls 1.0
 Control {
     id: root
 
-    // Expected roles: ensName, walletAddress, imageSource and amount
+    // Expected roles: ensName, walletAddress, imageSource, amount, selfDestructAmount and selfDestruct
     property var model
 
     property string tokenName
+    property bool isSelectorMode: false
+
+    signal selfDestructChanged()
```

Comment:
We really need to track changes for individual holders, not in the UI, since here it is only a total count shown in the button, but we need this information to pass it to the backend since it is needed for doing the end action.